aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -15,10 +17,27 @@ namespace Assets.Scripts.Wfc
     /// </summary>
```

Comment:
Also maybe add a little more descriptive summary here on what it does and how to use it